pegasus/sites.v3/code.org/public/administrators-alt.erb

```diff
@@ -249,7 +249,7 @@ Did you know the best way to encourage students to try CS is to tell them they
 	<summary style="font-size: 18px">Are Code.org’s courses aligned to any standards?</summary>
 	<p>
 	<br>
-<p>Yes! Specifically, our courses were written using both the K-12 Framework for Computer Science and the 2017 CSTA standards. Additionally, CS Principles was written to AP standards. View standards alignment for each of our courses here: <a href="https://curriculum.code.org/csf-20/standards/" target="_blank">CSF</a>, <a href="https://curriculum.code.org/csd-20/standards/" target="_blank">CSD</a>, <a href="https://curriculum.code.org/csp-20/standards/" target="_blank">CSP</a>.</p>
+<p>Yes! Specifically, our courses were written using both the K-12 Framework for Computer Science and the 2017 CSTA standards. Additionally, CS Principles was written to AP standards. View standards alignment for each of our courses here: <a href="https://curriculum.code.org/csf-20/standards/" target="_blank">CSF</a>, <a href="https://studio.code.org/courses/csd-2021/standards/" target="_blank">CSD</a>, <a href="https://studio.code.org/courses/csp-2021/standards/" target="_blank">CSP</a>.</p>
```

Comment:
Why aren't we updating the CSF standards link?

Reply:
There are a few instances of not updating the CSF standards link in this PR. Is that intentional?

Reply:
good question, I am not sure about that! One observation is that while courses have a landing page for standards, I'm not sure where exactly we'd point people for CSF standards, because the standards are now spread across multiple pages. I will follow up on this.